db/re/item_db.conf

```diff
@@ -21139,11 +21134,9 @@ item_db: (
 	Script: <"
 		bonus bVit,20;
 		bonus bMdef,10;
-		bonus2 bAddRaceTolerance, RC_All, 30;
-		bonus bUnbreakableShield,0;
-		/* When you receive Melee Physical damage, chance of casting Protective Light for 60 seconds. */
-		/* Cancels Stun, Sleep, Curse, Stone Curse, Poison, Blind, Silence, Bleeding, Chaos, and Frozen. */
-		/* Protects the user against such effects for a set duration. */
+		bonus2 bAddRaceTolerance,RC_All,30;
+		bonus3 bAutoSpellWhenHit,"ALL_RAY_OF_PROTECTION",1,50;
```

Comment:
this line should be commented
https://github.com/HerculesWS/Hercules/blob/master/db/re/skill_db.txt#L1038